### ISSUE TITLE
Multiple CRC Fields

### ISSUE THREAD

**Issue by flakjacket95** [digidotcom/python-suitcase#51]
I know this is very uncommon but, I think there is an issue with how multiple CRC fields are handled. They aren't all back-filled as appropriate. I think there is an omission somewhere in the CRC field handling section.

From a very quick perusal, I think that https://github.com/digidotcom/python-suitcase/blob/6d30405f749b9505e724e415681853426a99c2ee/suitcase/structure.py#L59 is incorrect and should be `checksum_data = field.packed_checksum(data)`

**Comment by posborne** [comment]
Thanks @flakjacket95.  Please go ahead and add a regression test and fix in a PR if you are able.

**Comment by flakjacket95** [comment]
@posborne I completely forgot to follow up on this, but, it's been well over a year. I did what I could as you asked above, minus the regression test which I did not have time to do. #52 has in it what I managed to get working, but, I have seen no progress in resolving this issue, and no furtherance of that PR, and yet the issue still remains in your codebase today.

I guess I'll close my issue since there is no clear interest in resolving it.
